applications/rag/main.tf

```diff
@@ -142,14 +148,14 @@ module "jupyterhub" {
   url_domain_name          = var.jupyter_url_domain_name
   members_allowlist        = var.jupyter_members_allowlist
 
-  depends_on = [module.kuberay-operator, module.gcs]
+  depends_on = [module.namespace, module.gcs]
```

Comment:
This makes way more sense, thank you. This will also improve apply time, since both kuberay-operator and jupyterhub take O(minutes) to deploy.